.gitignore

```diff
@@ -1 +1,4 @@
 .DS_Store
+
+*.swp
```

Comment:
One last request, mind removing the `.gitignore` change? Instead I recommend using [`git config --global  core.excludesfile ...`](https://stackoverflow.com/questions/7335420/global-git-ignore). For example, I have `.*.sw?` for vim files in `~/.gitignore` on my machine.

Reply:
Sure. Should I just revert that commit?